BUILD_MESON

```diff
@@ -15,36 +15,46 @@ Buidling Dinit via Meson
 =-=-=-=-=-=-=-=-=-=-=-=-
 
 Buidling Dinit is available via two "Build system": meson & make.
-Its file tells everything you need to buidling Dinit via meson.
-Bulding dinit via meson requires "meson" and a C++11 compiler (GCC version 4.9 and later, or Clang ~5.0 and later, should be fine)
+This file contains instructions for building Dinit using meson.
+Bulding dinit via meson requires "meson" (0.47.0 or later) and a C++11 compiler
+(GCC version 4.9 and later, or Clang ~5.0 and later, should be fine).
+
+
+Short version
+=-=-=-=-=-=-=
+
+Run "meson setup builddir && meson compile -C builddir". Your system type will hopefully be
+detected automatically and appropriate configuration chosen, and Dinit will be built. Continue
+reading instructions at "Running the test suite" or skip straight to "Installation".
+
+If this fails, or if you are cross-compiling, read the "long version" instructions.
+
+
+Long version
+=-=-=-=-=-=-
 
 Meson configures all you need to build Dinit. It detects your OS, detects your compiler & sets something based on your system information.
 By Default no action is required & meson configures everything needed to build Dinit.
 
 Note that the "eg++" or "clang++" package must be installed on OpenBSD as the default "g++"
 compiler is too old. Clang is a part of the base system in recent releases.
 
-in Meson you need to prepare "Buidling Dir". simply just type:
+in Meson you need to prepare "build directory". simply just type:
 
     meson setup builddir/
 
-This command configures "Building dir" in builddir/.
-You can set desired directory (but we suggest dont use dinit's directory such as build/ & src/ or ... as "Buidling dir").
-If everything goes smoothly this will prepare everything to build dinit, dinitctl, and optionaly the shutdown/reboot/halt utilitys by default.
-
-Everything is ready! you can compile Dinit with changing current directory to builddir/ & run:
-
-     meson compile
+This command configures "build directory" in builddir/. You can set desired
+directory (but we suggest don't use dinit's directory such as build/ & src/ or ... as "build dir").
+If everything goes smoothly this will prepare everything to build dinit, dinitctl
+and optionaly the shutdown/reboot/halt utilitys by default.
 
-Note: on old Mesons, you must use "ninja compile" instead of "meson compile".
-This command builds the dinit, dinitctl, and optionaly the shutdown/reboot/halt utilitys by default.
+Everything is ready! you can compile Dinit with
 
-Finally install them via this command:
+     meson compile -C builddir
 
-     meson install
-
-This command installs the programs. you can specify an alternate installation by setting the "DESTDIR" variable,
-eg "DESTDIR=/tmp/temporary-install-path meson install".
+Note: on Mesons older than 0.54, you need to use "ninja -C builddir" instead of
+"meson compile -C builddir". This command builds the dinit, dinitctl
```

Comment:
I think "This command builds ..." should be before the "Note:". It looks like the sentence is part of the note, but it isn't (right?)